lib/backend/k8s/k8s_fv_test.go

```diff
@@ -532,13 +533,45 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		})
 
 		By("Applying changes to a node", func() {
```

Comment:
Can we do another Get() on the node after the Apply and assert that the right fields are present?